attic/ceph-dev-container-only/config/definitions/ceph-dev-container-only.yml

```diff
@@ -138,4 +138,4 @@
               username: CONTAINER_REPO_USERNAME
               password: CONTAINER_REPO_PASSWORD
       - build-name:
-          name: "#${BUILD_NUMBER} ${BRANCH}, ${DISTROS}, ${ARCH}, ${FLAVOR}"
+          name: "#${{BUILD_NUMBER}} ${{BRANCH}}, ${{DISTROS}}, ${{ARCH}}, ${{FLAVOR}}"
```

Comment:
I find it really interesting git decided which brackets were the new ones randomly...

Reply:
the github view shows them all to me ?

Reply:
Look ARCH vs. FLAVOR

<img width="666" alt="image" src="https://github.com/user-attachments/assets/8621d2d5-cce9-4692-93de-5b2c2598b745" />